config/locales/sufia.en.yml

```diff
@@ -20,8 +20,8 @@ en:
             label_short:  "All"
             label_long:   "All of Sufia"
           my_files:
```

Comment:
@yinlinchen It might be best to change the translation key to works as well.  That way the code will make more sense.